ksuid/ksuid.py

```diff
@@ -129,5 +129,6 @@ def _inner_init(self, dt: datetime, payload: bytes) -> bytes:
     @property
     def timestamp(self) -> float:
         return (
-            int.from_bytes(self._uid[: self.TIMESTAMP_LENGTH_IN_BYTES], "big") / self.TIMESTAMP_MULTIPLIER
-        ) + EPOCH_STAMP
+            float(int.from_bytes(self._uid[: self.TIMESTAMP_LENGTH_IN_BYTES], "big") / self.TIMESTAMP_MULTIPLIER)
```

Comment:
```suggestion
            float(int.from_bytes(self._uid[: self.TIMESTAMP_LENGTH_IN_BYTES], "big")) / self.TIMESTAMP_MULTIPLIER
```
Otherwise it truncates the division to be an int, I believe.

Reply:
Good catch.